jibri/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM jitsi/base-java
+
+RUN \
+	apt-dpkg-wrap apt-get update \
+	&& apt-dpkg-wrap apt-get install -y curl gnupg \
+	&& curl -4s https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+	&& echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google-chrome.list \
+	&& apt-dpkg-wrap apt-get update \
+	&& apt-dpkg-wrap apt-get install -y jibri google-chrome-stable vim procps \
```

Comment:
You'll need to change the jitsi repo to unstable or this won't work. For the moment, all containers will use the stable repo except this one, because it's not yet available on stable.

Reply:
done